back/app/models/custom_field.rb

```diff
@@ -26,6 +26,7 @@
 #  select_count_enabled   :boolean          default(FALSE), not null
 #  maximum_select_count   :integer
 #  minimum_select_count   :integer
+#  random_option_ordering :boolean          default(FALSE), not null
```

Comment:
I wonder how desirable it is to have a survey where some options are in random order, while other options are not randomized. If it's not so desirable, perhaps it could also make sense to specify this on the phase level? Of course I understand it's a bit late to change this.

Reply:
Think it gives maximum flexibility and is more obvious having it on a field level. Maybe there could be a usability improvement we could make on the front end - if you have previously selected random then it defaults the next field to random